### ISSUE TITLE
Followed readme for install but can't add any content or ideas

### ISSUE THREAD

**Issue by danblundell** [linnovate/openideal#8]
going to {{site}}/node/add/idea causes this error:

"Fatal error: Call to undefined function i18n_taxonomy_allowed_values() in drupal/modules/taxonomy/taxonomy.module on line 1496"

**Comment by missourispartan** [comment]
Same here. I got the SAME exact error in my installation! I only see two tabs "Challenges" and "Ideas" on the generic Drupal UI, no OpenIdeaL UI. 

During site installation I get ""Fatal error: Call to undefined function i18n_taxonomy_allowed_values() in drupal/modules/taxonomy/taxonomy.module on line 1496"

**Comment by danblundell** [comment]
Yup! I can turn all the modules on in the drupal admin but there's no example content and the functionality to add content (ideas/challenges etc) doesn't work at all.

**Comment by missourispartan** [comment]
Got it to work! Had to use an older LAMP stack. Ubuntu 14.04 Server, Apache2, MySQL 5.5, and PHP5. Apparently it's not ported forward for MySQL 5.7 yet.

**Comment by missourispartan** [comment]
However, for our purposes, Drupal is not working so great. There is much of the text all over the place written in Arabic, and Drupal refues to change it to English (North American). I can't seem to access the HTML code to directly edit pages, either. May have to go with a different solution.
